static/sass/styles.scss

```diff
@@ -1367,3 +1365,14 @@ hr.p-separator.is-shallow {
 .u-image-separation {
   margin-top: $spv--small;
 }
+
+// utility to add a strong border to the top of a section
+.u-data-border--top {
```

Comment:
I don't think this is necessary, there's a [vanilla component](https://vanillaframework.io/docs/patterns/rule#highlighted) that does the same thing